marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MappingStepController.java

```diff
@@ -83,6 +84,20 @@
         return new ResponseEntity<>(body, headers, HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/{stepName}/newDoc", method = RequestMethod.GET)
+    @ApiOperation(value = "Returns an XML or JSON source document as a string of JSON")
+    @Secured("ROLE_readMapping")
+    public ResponseEntity<JsonNode> getNewDocument(@PathVariable String stepName, @PathVariable String docUri) {
+        // Step defines which database to pull the identified doc from.
+        final JsonNode stepNode = newService().getStep(STEP_DEFINITION_TYPE, stepName);
+        final String databaseName = stepNode.get("sourceDatabase").asText();
```

Comment:
Is it okay to hard-code "sourceDatabase" (part of the step schema)?  I didn't find it used anywhere else within marklogic-data-hub-central.

Reply:
No longer relevant given preceding answer from Ryan.